dockerfiles/build.sh

```
@@ -0,0 +1,86 @@
+fi
+
+__dockerfile="${1}"
+
+##
+#  Conditions to set the context for docker build.
+#   1. If the Dockerfile is part of a git repo, the set to repo root.
+#   2. If not part of git repo, then set to the directory of the Dockerfile.
+#
+__context_dir="$(dirname "$(realpath "${__dockerfile}")")"
+
+pushd "${__context_dir}" > /dev/null
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)